util/src/test/java/io/kubernetes/client/CopyTest.java

```diff
@@ -79,12 +82,13 @@ public void run() {
                   InputStream is = copy.copyFileFromPod(pod, "container", "/some/path/to/file");
                 } catch (IOException | ApiException e) {
                   e.printStackTrace();
+                } finally {
+                  s.release();
```

Comment:
i think we should release the semaphore after complete reading from the stream, isnt it?

Reply:
ah never mind. the test only verifies if the endpoint was requested.